projects/storefrontlib/src/cms-components/navigation/footer-navigation/footer-navigation.component.ts

```diff
@@ -31,7 +32,7 @@ export class FooterNavigationComponent {
   constructor(
```

Comment:
I think you're missing the template isFeatureLevel directive

Reply:
Nope, this one is using it here: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5010/files/574d423f8225b41fa09b6cbe05e4ee8001cb732f#diff-f384e86b61782ff3b7e666128a1a79b6R71